libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -150,7 +150,7 @@ assembly::Statement Parser::parseStatement()
 			expectToken(Token::Comma);
 			elementary = parseElementaryOperation();
 			if (elementary.type() != typeid(assembly::Identifier))
-				fatalParserError("Variable name expected in multiple assignemnt.");
+				fatalParserError("Variable name expected in multiple assignment.");
```

Comment:
Can you please pull this fix (together with its test) into a separate PR?

Reply:
Ok, will be pulled out.